.travis.yml

```diff
@@ -264,6 +264,11 @@ os: linux
     cache:
       directories:
         - "$HOME/.ccache"
+        - depends/built
+        - depends/sources
+        - depends/work
+        - depends/x86_64-linux-gnu
+        - depends/x86_64-unknown-linux.gnu
```

Comment:
Can you check the correctness of this path (line 271)?
My local build makes a folder called
`depends/x86_64-unknown-linux-gnu`
and our CI drops the `unknown-` from the folder name, hence the path of line 270 is the one actually used.

Reply:
Good point. 2 things here :
1) I bad copied the path
2) I believe the "unknown" derives from builds on Windows' WSL